frontend/src/components/Profile/ProfileImage.js

```diff
@@ -1,37 +1,97 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { Box, Typography } from '@mui/material';
 import Button from '@mui/material/Button';
 import useProfileImage from './useProfileImage';
 import uploadProfileImage from '../../requests/put/uploadProfileImage';
 import Alert from '@mui/material/Alert';
+import deleteProfileImage from '../../requests/delete/deleteProfileImage';
+import Modal from '@mui/material/Modal';
+import styles from './styles';
 
 function ProfileImage() {
 
     let profileImage = useProfileImage();
     const [uploadedFile, setUploadedFile] = useState(null);
-    const [successAlert, setSuccessAlert] = useState(false);
+    const [successAlert, setSuccessAlert] = useState(null);
     const [errorAlert, setErrorAlert] = useState(null);
+    const [modalOpen, setModalOpen] = useState(false);
+    const [disabled, setDisabled] = useState(false);
 
     const handleFileChange = (event) => {
         const selectedFile = event.target.files[0];
         setUploadedFile(selectedFile);
     };
 
+    const handleFileDelete = () => {
+        setDisabled(true);
+        deleteProfileImage()
+            .then(res => {
+                setSuccessAlert(res.message);
+                setErrorAlert(null);
+
```

Comment:
If user successfully deleted his profile image, user now has default user image. That default user image should be displayed after successful deletion instead of old user image. So after user successfully deleted his image default user image should be seen.